core/new-gui/src/app/workspace/component/property-editor/operator-property-edit-frame/operator-property-edit-frame.component.html

```diff
@@ -19,6 +19,47 @@
       nzTheme="outline"
       nzType="edit"></i>
   </button>
+  <i
+    *ngIf="currentOperatorId?.includes('PythonLambdaFunction')"
+    nz-icon
+    nz-popover
+    nzPopoverTitle="Python Lambda Function Instructions"
+    nzType="question-circle"
+    nzTheme="outline"
+    [nzPopoverContent]="PythonLambdaPopContent"></i>
+
+  <ng-template #PythonLambdaPopContent>
+    You can add a new column by:
+    <ul>
+      <li>Click on the blue <strong>"+"</strong> button</li>
+      <li>Select <strong>"Add New Column"</strong> in the drop-down list of the first field</li>
+      <li>Type the name of the new column</li>
+      <li>Select the attribute type</li>
+      <li>Type the expression using Python syntax</li>
```

Comment:
"Type the expression using Python syntax" -> "Typing in an expression using the Python syntax"